### PR TITLE
Add link to list of supported languages/lexers for code blocks

### PR DESCRIPTION
**Pull request by jugmac00**

### CODE REVIEW

File: reuse/links.txt [hunk 1]
```diff
@@ -21,3 +21,5 @@
 
 .. |lxd-sphinx-extensions| replace:: ``lxd-sphinx-extensions``
 .. _lxd-sphinx-extensions: https://github.com/canonical/lxd-sphinx-extensions
+
+.. _pygments documentation: https://pygments.org/languages/
```
Comment (ru-fu) [inline reuse/links.txt:25]:
```suggestion
.. _Pygments documentation: https://pygments.org/languages/
```

File: .wordlist.txt [hunk 2]
```diff
@@ -9,3 +9,4 @@ reStructuredText
 UI
 Intersphinx
 toctree
+pygments
```
Comment (ru-fu) [inline .wordlist.txt:12]:
```suggestion
Pygments
```

File: style-guide.rst [hunk 3]
```diff
@@ -90,7 +90,7 @@ To start a code block, either end the introductory paragraph with two colons (``
 In both cases, the code block must be surrounded by empty lines.
 
 When explicitly starting a code block, you can specify the code language to enforce a specific lexer, but in many cases, the default lexer works just fine.
-
+For a list of supported languages and their respective lexers, see the official `pygments documentation`_.
```
Comment (ru-fu) [inline style-guide.rst:93]:
```suggestion
For a list of supported languages and their respective lexers, see the official `Pygments documentation`_.
```

**Review by ru-fu — COMMENTED** [review]
Sorry, a bit of nitpicking. ;)

**Comment by jugmac00** [comment]
Hi @ru-fu, thanks for your suggestions. I have updated my PR. It looks like the second check fails. Unfortunately, either the link 404s or I do not have permissions to view it. Could you please advise what needs to be done to get this finally merged? Thank you!

**Review by ru-fu — APPROVED** [review]
The failure is due to an update to `myst-parser`, which changes how the directive options must be specified.

So unrelated to this PR. Let's get this change in, I'll fix the failure afterwards.

**Pull request merged**